conf/test_proteins.config

```diff
@@ -0,0 +1,18 @@
+/*
+ * -------------------------------------------------
+ *  Nextflow config file for running tests
+ * -------------------------------------------------
+ * Defines bundled input files and everything required
+ * to run a fast and simple test. Use as follows:
+ *   nextflow run nf-core/epitopeprediction -profile test_proteins
+ */
+
+params {
+  max_cpus = 2
+  max_memory = 6.GB
+  max_time = 4.h
+
+  // Input data
+  peptides = 'https://github.com/nf-core/test-datasets/raw/epitopeprediction/testdata/proteins/proteins.tsv'
```

Comment:
Shouldn't this be ```proteins = ...``` ? As far as I understand this is about testing the protein FASTA functionality?

Reply:
ups, thanks!